libnmstate/nm/applier.py

```diff
@@ -321,7 +321,12 @@ def prepare_proxy_ifaces_desired_state(ifaces_desired_state):
         )
         new_ifaces_desired_state.append(port_iface_desired_state)
         # The "visible" slave/interface needs to point to the port profile
-        iface_desired_state[MASTER_METADATA] = port_iface_desired_state[
+
+        # The OVS port does not have actual interface in kernel, but
+        # its length still limited by NM connection.interface-name.
+        # https://bugzilla.redhat.com/show_bug.cgi?id=1788432
+        # As workaround, just use the ovs-interface name.
+        iface_desired_state[MASTER_METADATA] = iface_desired_state[
```

Comment:
I do not understand why you "fix" it here and not at the source, i.e. in `_create_ovs_port_iface_desired_state`. Could you please explain?

Reply:
I want to the OVS port profile still have the name with prefix `ovs-port-`, just the `connection.interface` use the interface name.